core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -21,6 +21,15 @@
   <div class="protractor-test-conversation-input"
        angular-html-bind="activeCard.interactionHtml">
   </div>
+    <div ng-if="currentInteractionHints.length" style="text-align: right; padding: 8px">
+      <span tooltip="<[!isHintAvailable() ? 'Please try answering the question first.' : 'Click here for a hint!']>">
```

Comment:
Added text "Try thinking a bit more before asking for another hint!" for now. I did try the automatic tooltip/popover thing but it isn't working as expected, I'm not able to enable/disable tooltips based on events other that mouseover/click/focus events. Using JQuery would be an easy way to do this, but I'm trying to look for a more "angular"-style way of doing it.